src/garage/envs/multi_env_wrapper.py

```diff
@@ -80,12 +80,12 @@ def __init__(self,
         self._sample_strategy = sample_strategy
         self._num_tasks = len(envs)
         self._active_task_index = None
-        self._observation_space = None
         self._mode = mode
+        self._last_observation = None
         for i, env in enumerate(envs):
-            if not isinstance(env, GarageEnv):
-                envs[i] = GarageEnv(env)
-        super().__init__(envs[0])
+            if isinstance(env, str):
+                envs[i] = GymEnv(env)
```

Comment:
hmm i think this goes a little too far i suggest you only support instances of `Environment`